src/screens/Login/Login.tsx

```diff
@@ -105,15 +56,13 @@ const Login = () => {
     }
   }, [session.signingInStatus]);
 
-  useEffect(() => {
-    setErrors((state) => ({ ...state, password: errorReducer.message }));
-  }, [errorReducer.message]);
```

Comment:
Getting rid of this effect would mean you wouldn't be able to display error from the API on failed login like `Email already exists`, etc

Reply:
Not really, the Error Reducer handles any response from the API call.  Thanks why is line 87 rendering the error